wordpress.org/public_html/wp-content/plugins/wporg-gp-translation-suggestions/inc/class-translation-memory-client.php

```diff
@@ -132,22 +138,35 @@ public static function query( string $text, string $target_locale ) {
 		}
 
 		if ( empty( $result['matches'] ) ) {
-			return [];
+			return array();
 		}
 
-		$suggestions = [];
+		$suggestions = array();
 		foreach ( $result['matches'] as $match ) {
-			$suggestions[] = [
+			$suggestions[] = array(
 				'similarity_score' => $match['score'],
 				'source'           => $match['source'],
 				'translation'      => $match['text'],
 				'diff'             => ( 1 === $match['score'] ) ? null : self::diff( $text, $match['source'] ),
-			];
+			);
 		}
 
 		return $suggestions;
 	}
 
+	/**
+	 * Deletes a translation from translation memory.
+	 *
+	 * @param string $original_string    Original string.
+	 * @param string $translation_string Translation string.
+	 * @param string $locale_slug        Locale slug.
+	 * @param string $set_slug           Translation set slug.
+	 */
+	public static function delete( string $original_string, string $translation_string, string $locale_slug, string $set_slug ):bool {
+		// @todo Implement.
```

Comment:
Please implement this like this. It will require changing the function signature.
```suggestion
		$body = wp_json_encode(
			array(
				'token'    => WPCOM_TM_TOKEN,
				'source'       => array(
					'singular' => $original->singular,
					'plural' => $original->plural,
					'context' => $original->context,
				),
				'translation' => array(
					'singular' => $translation->translation_0,
					'plural' => $translation->translation_1,
					'locale'   => $locale,
				)
			)
		);

		$request = wp_remote_post(
			self::API_BULK_ENDPOINT,
			array(
				'method'    => 'DELETE',
				'timeout'    => 10,
				'user-agent' => 'WordPress.org Translate',
				'body'       => $body,
			)
		);
```